sdk/src/constants.ts

```diff
@@ -14,12 +14,16 @@ const DEPLOYMENTS: { [key in SupportedChainIds]: Deployment } = {
     chainName: "goerli",
     contractAddress: "0x822F17A9A5EeCFd66dBAFf7946a8071C265D1d07",
     graphName: "hypercerts-testnet",
+    graphBaseUrl: "https://api.thegraph.com/subgraphs/name",
```

Comment:
Got a thought when I looked at the Docker config. Instead of breaking open this config for the integration test, what do you think of adding a proxy to Docker that intercepts the https://api.thegraph...." calls and routes them to "http://localhost:..."

Reply:
Ah if we add a proxy we'd need to install a local cert for https. That part i definitely don't want to do, unless we do all the rewriting _before_ it gets sent over the wire.

Reply:
For now I'll update so we pass in the full path. This will greatly simplify this whole thing here haha